### ISSUE TITLE
Question: Model Bigger than a Structure Block

### ISSUE THREAD

**Issue by Markozeta** [RavinMaddHatter/Structura#35]
Not sure if this is a feature request or not, but it seemed to me that if I could stack my oversized structure blocks, the program and image itself may be able to handle images larger than a single structure block can handle.  If I specify an axis and a direction to stack upon, could the program in theory stack 2 or 3 in a single direction with ease?  If so then I'll rephrase this as a feature request.  If there is a technical reason this couldn't be done (for example the image textures must be less than 64 blocks) than I'll just use two armor stands and stack it with two texture packs.

**Comment by bigjock85** [comment]
I'm sure hatter said that the size is limited to the capabilities of the structure block, so 64x64x64. he is working (or wanting to work) on coloured armour stands so you could have multiple files loaded at once, so this may just take time.

**Comment by ForestOfGold** [comment]
While this would be nice in theory, colored armor stands are a better solution because the armor stands would stop being rendered from too far away.

**Comment by RavinMaddHatter** [comment]
To be clear there is a hard coded limit of 70ish blocks in minecraft, when standing next to the stand i can render blocks 1000+ blocks away, but when you get 70ish blocks away it completely disappears. 64x64x64 seemes to be the sweet spot, although if you put a -64 offset in the Y you can technically make things 128 tall in beta 1.1.1-preview

**Comment by RavinMaddHatter** [comment]
Closing this as a technical limitation of minecraft prevents simply making it bigger, i have a project for multiple channels though So we can make it work, just not in this way
